src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -142,6 +151,19 @@ module.exports = React.createClass({
             );
         }
 
+        if (this.props.mxEvent.getType() === 'm.room.message') {
```

Comment:
hm, i guess limiting it to `m.room.message` is okay for now, but this will need to change in future when arbitrary events are more comment